google/cloud/forseti/common/util/date_time.py

```diff
@@ -75,21 +76,53 @@ def get_utc_now_datetime():
     return datetime.utcnow()
 
 
-def get_utc_now_timestamp_human():
+def get_utc_now_timestamp_human(date=None):
     """Get a human formatted str representing utcnow()
 
+    Args:
+        date (datetime): A datetime object representing current time in UTC.
+
     Returns:
           str: A timestamp in the classes default timestamp format.
     """
-    utc_now = get_utc_now_datetime()
+    utc_now = date or get_utc_now_datetime()
     return utc_now.strftime(string_formats.DEFAULT_FORSETI_HUMAN_TIMESTAMP)
 
 
-def get_utc_now_timestamp():
+def get_utc_now_timestamp(date=None):
```

Comment:
qq: is the optional arg here for testing purpose?  If yes, I can find some existing examples of how to patch this.

Reply:
It's not just for testing. We also use it anywhere we already have a datetime object and just need it formatted.